docs/tutorials/validate_data.rst

```diff
@@ -187,6 +374,7 @@ Below is the default configuration for
             module_name: great_expectations.render.renderer.slack_renderer
             class_name: SlackRenderer
 
+We will show how to use the two most commonly used actions that are available to this operator:
 
 Save Validation Results
```

Comment:
I'm questioning whether demonstrating S3 is in scope for this tutorial. However this should not be addressed now.